gcloud-workstation-base/README.md

```diff
@@ -0,0 +1,64 @@
+<!-- markdownlint-disable MD041 -->
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD028 -->
+
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+[![pre-commit][pre-commit-shield]][pre-commit-url]
+[![taskfile][taskfile-shield]][taskfile-url]
+
+
+# gcloud-workstation-base
+
+This is basic config for cloud workstation image used for development under docker container with preinstalled commands.
+
+## ENV
+
+|                         | Description             |
+| :---------------------- | :---------------------- |
+
+| (\*) required
+
+## How to test
+
+build docker container
+
+```console
+docker build -t gcloud-workstation-base ./gcloud-workstation-base
+```
+
+run gcloud-workstation-base container
+
+```console
+docker run --rm -it --name gcloud-workstation-base gcloud-workstation-base
+```
+
+## Notice
+
+<!-- MARKDOWN LINKS & IMAGES -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+<!-- Links -->
+
+[pre-commit]: https://pre-commit.com/
```

Comment:
Most of these lines does not make sense

Reply:
Did you mean links? Will clean up them. I tried to use template readme, but wrong copy paste

Reply:
Check now please. Thanks